homework_1/fizzbuzz/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	fizzbuzz "code-cadests-2021/homework_1/fizzbuzz/fizzbuzz_test"
+	"flag"
+	"log"
+)
+
+
+func main() {
+	start := flag.Int("start", 1, "Start value for Fizz Buzz game")
+	end := flag.Int("end", 10, "End value for Fizz Buzz game")
+	flag.Parse()
+	fizzbuzzslice, err := fizzbuzz.Fizzbuzzgame(*start , *end)
+	if err != nil {
+		log.Fatal(err)
+	}
+	fizzbuzz.Printgameresults(fizzbuzzslice)
```

Comment:
Nice, I like how you've split the fizzbuzz logic into generating output and printing output. This can be further improved by having a fizzbuzz struct with methods, but you still need to learn that in the upcoming lecture so this solution is perfectly fine.